android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -200,6 +203,26 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
         }
       }
 
+      if (url.startsWith("intent://")) {
+        try {
+          Context context = view.getContext();
+          Intent intent = Intent.parseUri(url, Intent.URI_INTENT_SCHEME);
+          if (intent != null) {
+            view.stopLoading();
+            PackageManager packageManager = context.getPackageManager();
+            ResolveInfo info = packageManager.resolveActivity(intent, PackageManager.MATCH_DEFAULT_ONLY);
+            if (info != null) {
+              context.startActivity(intent);
+            } else {
+              String fallbackUrl = intent.getStringExtra("browser_fallback_url");
+              view.loadUrl(fallbackUrl);
+            }
+            return true;
+          }
+        } catch (URISyntaxException e) {
```

Comment:
Please add a comment `// NOP` on catch block.

This means that is expected behavior and not forgetting implementation.

Reply:
Also, Java linter sometimes warns of empty catch.